<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlagsPropertyGridWidget</class>
 <widget class="QWidget" name="FlagsPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>125</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>125</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>299</width>
     <height>124</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="visibleCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Visible</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="inputCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="clipContentsCheckbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip Contents</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial State:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="initialStateComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>85</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="recursiveVisibleCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Recursive Visible</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>visibleCheckBox</tabstop>
  <tabstop>inputCheckBox</tabstop>
  <tabstop>clipContentsCheckbox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
